docbrown/core/src/graph.rs

```diff
@@ -170,7 +170,7 @@ impl TemporalGraph {
     }
 
     pub fn edges_len(&self) -> usize {
```

Comment:
Is this function intended to be exposed to the user? I would change the name to out_edles_len, but if it is exposed to the user I would create another one instead

Reply:
TemporalGraph should not be exposed to the user, it's GraphDB that should be

Reply:
Ok, in that case I'm going to rename edges_len to out_edges_len in the internal classes to avoid confusion